examples/property_estimator/start_client.py

```diff
@@ -8,22 +8,43 @@
 from propertyestimator.utils import get_data_filename, setup_timestamp_logging
 
 
-def submit_calculation_to_server():
+def request_estimate_blocking():
     """Submit calculations to a running server instance"""
+    setup_timestamp_logging()
+
+    # Load in the data set of interest.
+    data_set = ThermoMLDataSet.from_file_list(get_data_filename('properties/single_density.xml'))
```

Comment:
Suggest `ThermoMLDataset.from_file_list` could just be called `.from_file()`, but accept either a single filename, an iterable of filenames, or a single `File`-like object (or iterable of `File`-like objects).

Reply:
Yeah I think what you suggest looks better.